web/src/components/tasks/business-formation/contacts/Signatures.test.tsx

```diff
@@ -47,59 +47,65 @@ describe("Formation - Signatures", () => {
   ];
 
   describe("starting legal structures", () => {
-    it.each([...legalStructureIds])("adds additional signer for %s", async (legalStructureId) => {
-      const page = await getPageHelper({ legalStructureId }, { signers: [] });
-      page.fillText("Signer 0", "Red Skull");
-      page.checkSignerBox(0, "signers");
-
-      page.clickAddNewSigner();
-      page.fillText("Signer 1", "V");
-      page.checkSignerBox(1, "signers");
-
-      await page.submitContactsStep();
-      expect(currentBusiness().formationData.formationFormData.signers).toEqual([
-        generateFormationSigner(
-          {
-            name: "Red Skull",
-            signature: true,
-          },
-          legalStructureId
-        ),
-        generateFormationSigner(
-          {
-            name: "V",
-            signature: true,
-          },
-          legalStructureId
-        ),
-      ]);
-    });
+    it.each([...legalStructureIds])(
+      "adds additional signer for legalStructure is %s",
```

Comment:
[pebble] Can we update the description to `"adds additional signer if legalStructure is %s"`

Reply:
done!